### PR TITLE
Feat: Support Keyboard Shortcuts from POS

### LINKED ISSUE #1439: Consultas Acciones del Punto de Venta
Question
Asking questions about use

## Question

Cómo se configura para que un Usuario pueda generar las acciones del Punto de venta?

* Asignar Vendedor
* Desasignar vendedor
* Ingreso de dinero
* Cierre de Caja
* Cierre de Caja Detallado

image

image

**Vendedores asignados en la Terminal PDV**
Tengo esta definición y no comprendo por que no veo esas opciones

image

Por otro lado por favor podrían aclarar que realiza las siguientes acciones?

* Asignar Vendedor
* Desasignar vendedor
* Ingreso de dinero
* Cierre de Caja
* Cierre de Caja Detallado

### LINKED ISSUE #1796: Atajos de Teclado desde POS Transcoding
## Bug report

1. No se visualiza botón de acción para definir "Atajos de Teclado" desde el POS en Transcoding
2. No se visualiza opción de "Nueva Orden" o "Pago" para definirle un Atajo (Esto no se veía en Legacy) 

La prioridad de esto es baja. simplemente lo dejo relevado

#### Screenshot or Gif（截图或动态图）
image

#### Other relevant information
- Your OS:
- Web Browser:
- Node.js version:
- adempiere-vue version: Transcoding

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif

https://github.com/solop-develop/frontend-core/assets/45974454/4ea840c7-6434-4dba-ad4e-3a7ef5cf2e21

### CODE REVIEW

File: src/components/ADempiere/Form/VPOS2/Options/salesOrder.vue [hunk 1]
```diff
@@ -570,6 +570,7 @@ export default defineComponent({
       },
       // setter
       set(show) {
+        console.log({ show })
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/salesOrder.vue]: please remove this console.log

File: src/store/modules/ADempiere/form/VPOS/index.js [hunk 2]
```diff
@@ -187,6 +198,46 @@ export default {
...
+      return new Promise(resolve => {
+        if (isEmptyValue(pointOfSales)) {
+          pointOfSales = getters.getVPOS
+        }
+        const { id } = pointOfSales
+        listCommandShortcut({
+          posId: id
+        })
+          .then(response => {
+            console.log({ ...response })
```
Comment (EdwinBetanc0urt) [inline src/store/modules/ADempiere/form/VPOS/index.js]: please remove this console.log

File: src/store/modules/ADempiere/form/VPOS/order.js [hunk 3]
```diff
@@ -374,7 +377,8 @@ export default {
       return new Promise(resolve => {
         const pos = getters.getVPOS
         if (isEmptyValue(order)) resolve({})
-        if (order.document_status.value === 'CO') {
+        console.log(order.document_status.value)
```
Comment (EdwinBetanc0urt) [inline src/store/modules/ADempiere/form/VPOS/order.js]: please remove this console.log

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/store/modules/ADempiere/form/VPOS/order.js (hunk 3 above)
Comment (elsiosanchez) [reply src/store/modules/ADempiere/form/VPOS/order.js]: remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe

File: src/store/modules/ADempiere/form/VPOS/index.js (hunk 2 above)
Comment (elsiosanchez) [reply src/store/modules/ADempiere/form/VPOS/index.js]: remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe

File: src/components/ADempiere/Form/VPOS2/Options/salesOrder.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS2/Options/salesOrder.vue]: remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**